database/postgres/postgres.go

```diff
@@ -15,7 +16,6 @@ import (
 	"github.com/frain-dev/convoy/pkg/log"
 	"github.com/jmoiron/sqlx"
 	_ "github.com/newrelic/go-agent/v3/integrations/nrpq"
```

Comment:
I don't think we need this import anymore either

Reply:
True